src/test/java/pl/javastart/euler2/Euler2Test.java

```diff
@@ -0,0 +1,22 @@
+package pl.javastart.euler2;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class Euler2Test {
+
+    private Euler2 euler2 = new Euler2();
+
+    @ParameterizedTest
+    @CsvSource(value = {"9, 10 ","10, 10","11, 10" ,"34, 44", "4000000, 4613732" })
```

Comment:
trochę formatowanie do poprawy (spacje)
dodatkowo pytanie skąd wiesz, że dla 4000000 poprawnym wynikiem jest 4613732? Testy służą do wykrywania błędów, czyli musisz mieć pewność, że wskazany wynik jest faktycznie poprawny. Nie możesz wyciągnąć wniosku, że jeżeli wcześniejsze testy, które zweryfikowałeś np. na kartce, albo w excelu są poprawne, to dla dowolnej liczby Jeżeli nie jesteś w stanie zweryfikować swojego wyniku w sposób inny niż przy pomocy tego programu, to nie uwzględniałbym takiego testu